NetscapeBookmarkParser.php

```diff
@@ -45,9 +42,9 @@ public function __construct(
         bool $keepNestedTags = true,
         ?array $defaultTags = [],
         $defaultPub = '0',
-        string $logDir = null,
         bool $normalizeDates = true,
-        string $dateRange = '30 years'
+        string $dateRange = '30 years',
+        LoggerInterface $logger = null
```

Comment:
PHPDoc should be updated.

Reply:
Hmm, OK... Well, sometimes PHPDoc are counterproductive... We have type hints, and a default value. And now that PHP8 brings named attributes, it actually makes sense to avoid having to maintain code in two places. By default Symfony linter removes PHPDocs that adds no value. Maybe they will become deprecated at some point I don't know.

Reply:
I agree with you, and it's usually what I do in other new projects. But with this PHP compatibility, we don't have typed attributes yet, and in any case the PHPDoc shouldn't be outdated.
Thanks for the change.